lib/Mojolicious/Plugin/RevealJS.pm

```diff
@@ -41,7 +41,7 @@ sub _include_code {
     % require Mojo::Util;
     % my $file = stash 'revealjs.private.file';
     <pre><code class="<%= stash('language') // 'perl' %>" data-trim>
-      <%= Mojo::Util::slurp(app->home->rel_file($file)) =%>
+      <%= Mojo::File::slurp(app->home->rel_file($file)) =%>
```

Comment:
Mojo::File::slurp is not a function, it is a method. `app->home->rel_file($file)->slurp`